firmware-management/swupdate-handlers/arm-handler-common.h.in

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+*/
+
```

Comment:
Almost missed it! We should have include guards in header files.
```C
#ifndef swupdate_handlers_arm_handler_common_h_
#define swupdate_handlers_arm_handler_common_h_
...
#endif // swupdate_handlers_arm_handler_common_h_
```

rootfs-handler.h too.

Reply:
Yeah I spotted that myself this morning and have added them to my local clone. Will be changed in the next commit.

Reply:
:+1: